source/runbooks/set-up-macie.html.md.erb

```diff
@@ -0,0 +1,89 @@
+
+Set up a [Macie account](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/macie2_account)
+
+Creating a [Macie job](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/macie2_classification_job)
+
+Below there is an example of how the items were set up in Cooker following the instructions listed here.
+
+## Example Terraform code
+
+There is one pre-requisite, for the code shown here, to allow this to work and run successfully. Make sure that the role for **MemberInfrastructureAccess has ace macie2:** set. If this has not been set up correctly (by the Modernisation Platform Team) the terraform listed below will not work. If this is amended in the future, so that it is enabled globally, this comment will be removed. 
+
+The full details of the "finding_publishing_frequency" and "job_type" can be seen in the terraform links listed above.
+
+Full details of this creation, including information on the likes of job type, are shown in the code (macie.tf) in the modernisation-platform-environments repo for cooker. The appropriate environment account reference should be used rather than the "account-code for this environment" shown below.
+
+```
+# Create macie account
+# When logging in as administrator this is not needed but may be for our customers - yet to be fully determined.
+# Account name is MacieAccount. Publishes data every hour (choice are FIFTEEN_MINUTES, ONE_HOUR or SIX_HOURS).
+
+resource "aws_macie2_account" "MacieAccount" {
+  finding_publishing_frequency = "ONE_HOUR"
+  status                       = "ENABLED"
+}
+
+# Now create a job
+# Job type options are ONE-TIME and SCHEDULED. If scheduled is chosen you need to select a frequency using schedule_frequency of daily_schedule, weekly_schedule or monthly_schedule
+# The buckets list should be replaced but a list of buckets to examine. This could be limited to those in the account you are connected to or others to which you have access.
+# Below there is a limited list of those from example shown.
+# There are many other job options available and these should be looked at in terrform [job creation](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/macie2_classification_job)
+
+
+resource "aws_macie2_classification_job" "test" {
+  job_type = "ONE_TIME"
+  name     = "<an appropriate job name>"
+  s3_job_definition {
+    bucket_definitions {
+      account_id = "<account-code for this environment>"
+      buckets    = [
+      "<list of buckets>",
+      "<separated by commas>"
+      ]
+    }
+  }
+  depends_on = [aws_macie2_account.MacieAccount]
+}
+```
+
+# Set up Macie jobs
```

Comment:
This is manual steps again which should be remove from the document, we do not want users doing things manually and they won't be able to do this in any environment other than sandbox.

Reply:
These need to be created manually unfortunately. 
Users will need to amend these based on their requirements. 
The account code bit has since been amended (and comments added about this) but other manual changes may be needed.